content/servicemesh/routing.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Intelligent Routing"
+date: 2018-11-13T21:49:32+09:00
+weight: 50
+draft: false
+---
+
+### Intelligent Routing
+
+> Deploying a microservice-based application in an Istio service mesh allows one to externally control service monitoring and tracing, request (version) routing, resiliency testing, security and policy enforcement, etc., In a consistent way across the services, for the application as a whole.
+
+Before you can use Istio to control the Bookinfo version routing, you need to define the available versions, called <span style="color:orange">*`subsets`*</span>, in destination rules
+
+```
+kubectl apply -f samples/bookinfo/networking/destination-rule-all.yaml
+
+kubectl get destinationrules -o yaml
+```
+
+To route to one version only, you apply virtual services that set the default version for the microservices. In this case, the virtual services will route all traffic to <span style="color:orange">**reviews:v1**</span> of each microservice.
+
+```
+kubectl apply -f samples/bookinfo/networking/virtual-service-all-v1.yaml
+
+kubectl get virtualservices reviews -o yaml
```

Comment:
Can you define what is virtualservices and go into details of how does it make use of subsets?

Reply:
added some explanation about yaml output